diff --git a/nginx/conf.d/default.conf b/nginx/conf.d/default.conf
index 34aeb9a..de11a1e 100644
--- a/nginx/conf.d/default.conf
+++ b/nginx/conf.d/default.conf
@@ -10,6 +10,17 @@ server {
         index  index.html index.htm;
     }
 
+    location /mruby {
+        mruby_content_handler_code '
+          if server_name == "NGINX"
+            Server = Nginx
+          elsif server_name == "Apache"
+            Server = Apache
+          end
+          Server::rputs "Hello #{Server::module_name}/#{Server::module_version} world!"
+        ';
+    }
+
     #error_page  404              /404.html;
 
     # redirect server error pages to the static page /50x.html
diff --git a/nginx/nginx.conf b/nginx/nginx.conf
index e4bad8d..ab35916 100644
--- a/nginx/nginx.conf
+++ b/nginx/nginx.conf
@@ -5,6 +5,7 @@ worker_processes  1;
 error_log  /var/log/nginx/error.log warn;
 pid        /var/run/nginx.pid;
 
+load_module /usr/lib/nginx/modules/ngx_http_mruby_module.so;
 
 events {
     worker_connections  1024;
