diff --git a/nginx/conf.d/default.conf b/nginx/conf.d/default.conf
index 34aeb9a..4e73d0e 100644
--- a/nginx/conf.d/default.conf
+++ b/nginx/conf.d/default.conf
@@ -10,6 +10,13 @@ server {
         index  index.html index.htm;
     }
 
+    location /mruby {
+        mruby_content_handler_code '
+          Nginx.rputs "Hello world!\n"
+          Nginx.rputs "#{Nginx.module_name}/#{Nginx.module_version}\n"
+        ';
+    }
+
     #error_page  404              /404.html;
 
     # redirect server error pages to the static page /50x.html
diff --git a/nginx/nginx.conf b/nginx/nginx.conf
index e4bad8d..510aa85 100644
--- a/nginx/nginx.conf
+++ b/nginx/nginx.conf
@@ -5,6 +5,8 @@ worker_processes  1;
 error_log  /var/log/nginx/error.log warn;
 pid        /var/run/nginx.pid;
 
+load_module modules/ndk_http_module.so;
+load_module modules/ngx_http_mruby_module.so;
 
 events {
     worker_connections  1024;
